src/NewsletterBundle/Resources/config/services.xml

```diff
@@ -5,6 +5,10 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
+        <instanceof id="\IViewMapper" autoconfigure="false">
```

Comment:
This is the strangest of the three autoconfiguration variants (here autoconfigure=false).

instanceof and configuration cannot be mixed.